nixos/modules/virtualisation/podman/default.nix

```diff
@@ -9,8 +9,7 @@ let
     extraPackages = cfg.extraPackages
       # setuid shadow
       ++ [ "/run/wrappers" ]
-      # include pkgs.zfs by default in the wrapped podman used by the module so it is cached
```

Comment:
IIRC this should not be cached. Also this adds unnecessarily zfs to every system that uses podman which adds 90MB closure.

Reply:
Please suggest an alternative that will also be cached so we can avoid building from source if zfs is enabled.

Reply:
*this would not be cached.

I think the easiest way it to expose a package to top-level.

Reply:
> *this would not be cached.

I believe it would be as the podman test is a channel blocker.

> I think the easiest way it to expose a package to top-level.

Yes, already thought of that and decided against it, don't want the user to have to pick the appropriate package.

Reply:
I don't think things get uploaded that where build when running the tests.

We don't need to make people pick that package, we just want it to be cached and then we can use it in the module.

Reply:
It is cached: /nix/store/iyn7r3k5csm8kwbw1nddmgpqxncfkahb-podman-4.4.1

Reply:
As the tests are cached we can have a tests with and without zfs so both are cached. https://github.com/NixOS/nixpkgs/pull/218355